generators/server/templates/gradle.properties.ejs

```diff
@@ -62,7 +62,7 @@ cassandraDriverVersion=4.14.1
 <%_ } _%>
 
 # gradle plugin version
-jibPluginVersion=<%= JIB_VERSION %>
+jibPluginVersion=<%- javaDependencies['jib-maven-plugin'] %>
```

Comment:
We are using the maven plugin version at gradle, this looks weird but should be easier to track if the version diverge.